beetsplug/ftintitle.py

```diff
@@ -35,9 +35,15 @@ def split_on_feat(artist):
         return tuple(parts)
 
 
-def contains_feat(title):
+def contains_feat(title, for_artist=True):
     """Determine whether the title contains a "featured" marker."""
-    return bool(re.search(plugins.feat_tokens(), title, flags=re.IGNORECASE))
+    return bool(
+        re.search(
+            plugins.feat_tokens(for_artist=for_artist),
+            title,
+            flags=re.IGNORECASE,
+        )
+    )
```

Comment:
Seems like `contains_feat` is only ever used with `for_artist=False`. In such case, I think it's best to remove it from the function params and hardcode it:

```python
def contains_feat(title):
    """Determine whether the title contains a "featured" marker."""
    return bool(
        re.search(
            plugins.feat_tokens(for_artist=False),
            title,
            flags=re.IGNORECASE,
        )
    )
```

Additionally, given that the function is called `contains_feat` I reckon using `for_artist=True` would be a bit misleading, since it matches not only featuring but also multiple artists.

Reply:
That is true. I am not the author of the original code, so I was not sure whether there is a purpose behind the `for_artist` keyword. But I agree that it is not needed in the current version.